src/__tests__/updater.js

```diff
@@ -0,0 +1,133 @@
+import { enhanceReducer } from "../updater";
+import { OFFLINE_STATUS_CHANGED, RESET_STATE } from "../types";
+import {
+  OFFLINE_COMPLETE_RETRY,
+  OFFLINE_SCHEDULE_RETRY,
+  PERSIST_REHYDRATE,
+  OFFLINE_BUSY
+} from "../constants";
+
+const initialState = {
+  busy: false,
+  lastTransaction: 0,
+  online: false,
+  outbox: [],
+  retryCount: 0,
+  retryScheduled: true,
+  netInfo: {
+    isConnectionExpensive: null,
+    reach: "NONE"
+  }
+};
+
+const config = {
+  queue: {
+    enqueue: jest.fn(),
```

Comment:
I think you're safe to use our default config queue methods here so we can correctly test that `offline.outbox` gets updated correctly.

Also, we should probably have a couple of tests which assert that we can pass in a custom enqueue/dequeue, in which case it's probably safe to mock them this way/test the new queue funcs change the outbox in the desired way.